Doc/library/datetime.rst

```diff
@@ -122,6 +122,14 @@ Available Types
 
    .. versionadded:: 3.2
 
+.. class:: IsoCalendarDate
```

Comment:
Perhaps @taleinat will disagree with me, but if we do make this switch (and as I've said in the BPO thread, I'm not in favor of it), I consider the name and details of the return type class to be a bit of an implementation detail.

Practically speaking we would need to document that it is a `namedtuple` of some sort and what its attributes are, but ideally this would be a purely private class. I would never want to see code like `isinstance(x, datetime.IsoCalendarDate)` or `x = IsoCalendarDate(2019, 1, 1)`.

As such, if you document it anywhere it should probably not be referred to by name and just document its properties in the `isocalendar` documentation.